Xamarin.PropertyEditing.Mac/PropertyTableDataSource.cs

```diff
@@ -3,34 +3,80 @@
 using System.Linq;
 using AppKit;
 using Foundation;
-using Xamarin.PropertyEditing.Mac.ViewModels;
 using Xamarin.PropertyEditing.ViewModels;
 
 namespace Xamarin.PropertyEditing.Mac
 {
-	public class PropertyTableDataSource : NSOutlineViewDataSource
+	internal class PropertyTableDataSource
+		: NSOutlineViewDataSource
 	{
-		internal MacPanelViewModel ViewModel { get; private set; }
```

Comment:
There wasn't any real reason for it. There's no reuse case, it simply moves code from where it was actually being used to another class and it's not really complicated enough to warrant tests.